pkg/ttnpb/field_mask_validation.go

```diff
@@ -497,7 +497,6 @@ var AllowedFieldMaskPathsForRPC = map[string][]string{
 		"ids",
 		"ids.application_ids",
 		"ids.application_ids.application_id",
-		"ids.dev_addr",
```

Comment:
@htdvisser  can you confirm that this is not passed by CLI anymore? If so, I'm good with it.
FTR, AS does not allow it to be set either.

Reply:
The CLI uses `AllowedFieldMaskPathsForRPC` to determine what fields to set/get, so that should be fine.

Reply:
Isn't this for setting ABP device addresses?

Reply:
> Isn't this for setting ABP device addresses?

The console sets it in `session.dev_addr`

Reply:
Right